tar/multitape/multitape_write.c

```diff
@@ -423,8 +423,8 @@ writetape_open(uint64_t machinenum, const char * cachedir,
 	if ((d->dryrun == 0) && multitape_cleanstate(cachedir, machinenum, 0))
 		goto err4;
 
-	/* Get sequence number. */
-	if (multitape_sequence(cachedir, lastseq))
+	/* If we're using a cache, get the sequence number. */
+	if ((cachedir != NULL) && (multitape_sequence(cachedir, lastseq)))
```

Comment:
I'm wondering if this should be (cachedir != NULL) or if (d->dryrun == 0) would make more sense.  Also, are we going to end up using lastseq without it being initialized?